django/forms/models.py

```diff
@@ -642,16 +665,34 @@ def save(self, commit=True):
         as necessary, and returns the list of instances.
         """
         if not commit:
-            self.saved_forms = []
-
-            def save_m2m():
-                for form in self.saved_forms:
-                    form.save_m2m()
-            self.save_m2m = save_m2m
+            self._update_save_m2m()
         return self.save_existing_objects(commit) + self.save_new_objects(commit)
 
     save.alters_data = True
 
+    def get_updated_models(self):
+        """
+        Saves each ``form``'s cleaned_data into the corresponding new or modified
+        model instances, and returns the list of instances.
+
+        Changes to the models are never saved to database.
+
+        This is a utility method parallell to ``BaseModelForm.get_upodated_models()``
```

Comment:
Groan, thanks. The tests don't catch that.

It's actually get_updoated_models => get_updated_model, I meant to say the same thing as in my answer to your note to 673 above.